File: fixtures/axt-outline/langs/Service.php
<?php

namespace Example\Service;

// User service.
class Service
{
    // Load one user.
    public function load(string $id): User
    {
        return new User($id);
    }

    private function cacheKey(string $id): string
    {
        return $id;
    }
}

interface UserPort
{
    public function find(string $id): User;
}

File: fixtures/axt-outline/langs/Service.java
package example.service;

// User service.
public class Service {
    // Load one user.
    public User load(String id) {
        return new User(id);
    }

    private String cacheKey(String id) {
        return id;
    }
}

interface UserPort {
    User find(String id);
}
